src/templates/static/platforms.jsx

```diff
@@ -121,25 +121,29 @@ const Platform = () => (
         <div className='fill-bg-color gr-padding-30 mobile-hide' data-show='-eucountry'>
             <div className='container gr-padding-30 gr-child'>
                 <h2 className='primary-color center-text'>
-                    {it.L('Deriv — an all-new trading experience')}
+                    {it.L('Introducing Deriv.com')}
                 </h2>
+                <p className='center-text gr-child'>{it.L('A rebranding of Binary.com to an all-new trading experience')}</p>
```

Comment:
please use it.website_name instead of Binary.com

Reply:
for Binary.com can use website_name but for Deriv.com can type Deriv.com?

Reply:
yes Deriv.com can remain